extensions/common/iam/identity-trust/identity-trust-core/src/main/java/org/eclipse/edc/iam/identitytrust/core/IdentityAndTrustExtension.java

```diff
@@ -87,6 +94,10 @@ public class IdentityAndTrustExtension implements ServiceExtension {
     private String issuerId;
     @Setting(description = "The period of the JTI entry reaper thread in seconds", defaultValue = DEFAULT_CLEANUP_PERIOD_SECONDS + "", key = "edc.sql.store.jti.cleanup.period")
     private long reaperCleanupPeriod;
+
+    @Setting(description = "If set enable the dcp v0.8 namespace will be used", key = "edc.dcp.v08.enabled", required = false, defaultValue = "false")
+    private boolean enableDcpV08;
```

Comment:
minor nit: i would call that property `edc.dcp.v08.forced`, since we're "forcing" (rather than "enabling) the v0.8 namespace.